includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -2063,4 +2086,25 @@ public function get_active_loan_summary() : array {
 	public function get_loans() : array {
 		return $this->request( [], self::CAPITAL_API . '/loans', self::GET );
 	}
+
+	/**
+	 * Hashes customer data for the fraud prevention.
+	 *
+	 * @param string $data The data you want to hash.
+	 *
+	 * @return string Hashed data.
+	 */
+	public function hash_data_for_fraud_prevention( $data ) {
+		return hash( 'sha512', $data );
+	}
+
+
+	/**
+	 * Checks if fraud prevention is enabled for this store.
+	 *
+	 * @return bool True if store has fraud prevention enabled.
+	 */
+	public function is_fraud_prevention_enabled_for_store() {
+		return false;
```

Comment:
This is a way of making this behind a "feature flag" :)

Reply:
With https://github.com/Automattic/woocommerce-payments/pull/3994 merged, I believe we can refactor this.